### PR TITLE
Rename `normalized_chi()` to `fei()`

### LINKED ISSUE #474: Rename normalized Chi to Waw / Fei?
Following a scholarly discussion on the [bird app](https://twitter.com/mattansb/status/1568328501056663552?t=a9lhCzl_nn9odmvMQnSlYA&s=19) with @bwiernik , I suggest renaming the nameless effect size to either:

- Waw (ו)
- Fei (פ)

The former is related to v and w, which are other Chisq effect sizes, and is pronounced like word "wow". The latter sounds like phi and maybe looks better in print?

### PR DESCRIPTION
**Pull request by mattansb**
Closes #474 

The פ should render fine in the docs, right?

### CODE REVIEW

**Comment by mattansb** [comment]
Hmm... I'll take that as a "no".

Any ideas @bwiernik @strengejacke @IndrajeetPatil ?

```
❯ checking PDF version of manual ... WARNING
  LaTeX errors when creating PDF version.
  This typically indicates Rd problems.
  LaTeX errors found:
  ! Package inputenc Error: Unicode character פ (U+05E4)
  (inputenc)                not set up for use with LaTeX.
```

**Comment by bwiernik** [comment]
Does using the `\u` Unicode escape work?

**Comment by mattansb** [comment]
Currently it's always in `\eqn{פ}`.
I'm gonna replace it with `XX` just to see if all of the errors are related to that.

**Comment by mattansb** [comment]
- `ubuntu-latest (release)` + `ubuntu-latest (devel)` fails due to not being able to build XML
- `ubuntu-latest (oldrel-2)` + `ubuntu-latest (oldrel-3)` are failing due to running code in examples that is in `\donttest{}`.

@IndrajeetPatil, any idea why these are happening?

**Comment by IndrajeetPatil** [comment]
This is beyond our control, unfortunately.

https://github.com/r-lib/actions/issues/559

**Comment by mattansb** [comment]
Ah okay :/

Any idea why an example is running in `\donttest{}`? (I hate these tags...)

**Comment by bwiernik** [comment]
CRAN tests examples in donttest (for some inexplicable reason). If you want them not to run ever, use dontrun

**Comment by bwiernik** [comment]
You can't use the actual symbol. Only ASCII is allowed on CRAN. You need to use latex macros in eqn or the Unicode escape numbers. For the PDF, it might be the case that we just can't use the symbols if there isn't a built in latex macro we can use in eqn. In that case, have it conditionally display the symbol in HTML and plain text (via the Unicode escape) and just the fei spelling in Latex.

**Comment by mattansb** [comment]
I'm having some problems with the escape numbers - they don't seem to be doing anything:
```
Convert between Chi square (\eqn{\chi^2}), phi (\eqn{\phi}), Cramer's V,
Cohen's *w*, Fei (\u05e4) and Pearson's *C* for contingency tables or
goodness of fit.
```

Is just rendered as:

image

What am I doing wrong? 🤔

File: R/convert_stat_chisq.R [hunk 1]
```diff
@@ -1,10 +1,9 @@
 #' Conversion Chi-Squared to Phi or Cramer's V
 #'
 #' Convert between Chi square (\eqn{\chi^2}), phi (\eqn{\phi}), Cramer's V,
-#' Cohen's *w*, Fei (*פ*) and Pearson's *C* for contingency tables or
```
Comment (IndrajeetPatil) [inline R/convert_stat_chisq.R]: What symbol is that?

Is that Hebrew letter _pe_?
https://en.wiktionary.org/wiki/%D7%A4

Comment (mattansb) [reply R/convert_stat_chisq.R]: Yes - the letter can be pronounced as P or as F; when pronounced as F, the letter is called Fe; in Hebrew that last vowel usually sounds more like the "ei" diphthong, and so - Fei.

Thanks' for coming to my Hebrew lesson (:

**Comment by bwiernik** [comment]
I'll try it out

**Comment by bwiernik** [comment]
The Writing R Extensions docs make it sound like including UTF-8 characters in the Latex version should be possible (https://cran.r-project.org/doc/manuals/R-exts.html#Encoding), but I can't figure out what to do there. 

So, for now, I just set it to use the proper symbols in HTML and text, but Fei in PDF (ironic considering that Writing R Extensions says "Mathematical formulae should be set beautifully for printed [PDF] documentation")

**Comment by bwiernik** [comment]
Can you share the links to the StackOverflow examples?

**Comment by bwiernik** [comment]
> welcome Fei!

The title of the paper should totally start with "phi, fei, fo, fum…"

**Comment by mattansb** [comment]
https://stackoverflow.com/questions/43850229/is-it-possible-to-write-package-documentation-using-non-ascii-characters-with-ro

**Pull request merged**